docs/en/ingest-management/release-notes/release-notes-8.4.asciidoc

```diff
@@ -27,6 +27,47 @@ Also see:
 [[release-notes-8.4.1]]
 == {fleet} and {agent} 8.4.1
 
+[discrete]
+[[known-issues-8.4.1]]
+=== Known issues
+
+// tag::credentials-error[]
+.Credentials error prevents {agent} from ingesting AWS logs
+[%collapsible]
+====
+
+*Details*
+
+{agent}s configured to use the AWS S3 and SQS integrations may return the
+following error:
+
+[source,shell]
+----
+sqs ReceiveMessage failed: operation error SQS: ReceiveMessage, https response
+error StatusCode: 403, RequestID: cb57783a-505f-5099-9160-23b8eea8ddbb,
+api error SignatureDoesNotMatch: Credential should be scoped to a valid region. 
+----
+
+This error was introduced by a breaking change in the AWS library.
+
+*Impact* +
+
+{agent} is unable to ingest AWS logs and some metrics. To resolve this error:
+
+* If you are using the default domain `amazonaws.com`, upgrade the AWS
```

Comment:
Do we need to mention explicitly that the custom endpoint configuration must be removed, or will the package upgrade just remove it?

Reply:
The package upgrade will remove it 🙂